src/webextension/manage/containers/current-item.js

```diff
@@ -18,28 +18,35 @@ function unflattenItem(item, id) {
   return {
     id,
     title: item.title,
+    origins: item.origin ? [item.origin] : [],
     entry: {
       kind: "login",
       username: item.username,
       password: item.password,
+      notes: item.notes,
     },
   };
 }
 
 function flattenItem(item) {
   return {
     title: item.title,
+    origin: item.origins[0] || "",
```

Comment:
Nice, so if I understand correctly: we're just capturing one string for now and putting it into the array to match the expected datastore model.

Reply:
yes, exactly.  We can expand this later.